### PR TITLE
chore: remove double semi-colon

### PR DESCRIPTION
**Pull request by albertosantini**
### Description 
Remove a double semi-colon.

### Testing done 
The file is compiled.

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksql-functional-tests/src/test/java/io/confluent/ksql/test/planned/TestCasePlanLoader.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
 import java.io.IOException;
 import java.nio.charset.Charset;
 import java.nio.file.Files;
-import java.nio.file.Path;;
+import java.nio.file.Path;
```
Comment (vcrfxia) [inline ksql-functional-tests/src/test/java/io/confluent/ksql/test/planned/TestCasePlanLoader.java]: Looks like this import is actually unused. Perhaps we can remove the line entirely?

Comment (albertosantini) [reply ksql-functional-tests/src/test/java/io/confluent/ksql/test/planned/TestCasePlanLoader.java]: @vcrfxia Well, I am newbie in Java, but I see 

https://github.com/confluentinc/ksql/blob/78b46ec5152b78d80bce1bc7b95aa415feacb230/ksql-functional-tests/src/test/java/io/confluent/ksql/test/planned/TestCasePlanLoader.java#L86

or

https://github.com/confluentinc/ksql/blob/78b46ec5152b78d80bce1bc7b95aa415feacb230/ksql-functional-tests/src/test/java/io/confluent/ksql/test/planned/TestCasePlanLoader.java#L115

and many other references.

If I remove that line, I see many errors in VSCode.

What am I missing?

Comment (spena) [reply ksql-functional-tests/src/test/java/io/confluent/ksql/test/planned/TestCasePlanLoader.java]: @albertosantini you're seeing an old version of this file. try to rebase your code and you'll see those methods do not exist anymore.

Comment (albertosantini) [reply ksql-functional-tests/src/test/java/io/confluent/ksql/test/planned/TestCasePlanLoader.java]: Ok. Thanks. I disgregard the PR. Finally I submit a new one.

**Pull request closed without merging**